MAUI/NumericEntry/Basic-Features.md

```diff
@@ -268,4 +268,114 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
 
 The following image illustrates the result of the above code:
 
-![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
\ No newline at end of file
+![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
+
+## Set the compute to percentage in SfNumericEntry
+
+When the [SfNumericEntry](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html) is in percentage format, the value can be displayed in two ways as follows:
+
+`Value`: Displays the actual value with percentage symbol.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry x:Name="numericEntry" 
+                        CustomFormat="p" 
+                        Value="1000"  
+                        PercentDisplayMode="Value">
+</editors:SfNumericEntry>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry= new SfNumericEntry();
+SfNumericEntry sfNumericEntry = new SfNumericEntry();
+sfNumericEntry.Value = 1000;
+sfNumericEntry.CustomFormat = "p";
+sfNumericEntry.PercentDisplayMode = PercentDisplayMode.Value;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry PercentDisplayMode](GettingStarted_images/percentdisplaymode_value.png)
+
+`Compute`: Displays the computed value with percentage symbol.
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfNumericEntry x:Name="numericEntry" 
+                        CustomFormat="p" 
+                        Value="1000"  
+                        PercentDisplayMode="Value">
+</editors:SfNumericEntry>
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry= new SfNumericEntry();
+sfNumericEntry.Value = 1000;
+sfNumericEntry.CustomFormat = "p";
+sfNumericEntry.PercentDisplayMode = PercentDisplayMode.Compute;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry PercentDisplayMode](GettingStarted_images/percentdisplaymode_compute.png)
+
+N> Default value of PercentDisplayMode is `Compute`.
+
+## Set the maximum number of decimal digits in SfNumericEntry
+
+The maximum number of digits to be displayed after the decimal point can be specified by using the `MaximumNumberDecimalDigits` property.
+
+N> The `MaximumNumberDecimalDigits` property can be provided with positive value only. Default value of MaximumNumberDecimalDigits is `2`.
```

Comment:
@SivaranjithN  Add the MaximumNumberDecimalDigits not working for custom format details also